libraries/Wire/Wire_nRF52.cpp

```diff
@@ -401,17 +401,34 @@ void TwoWire::onService(void)
   }
 }
 
-TwoWire Wire(NRF_TWIM1, NRF_TWIS1, SPIM1_SPIS1_TWIM1_TWIS1_SPI1_TWI1_IRQn, PIN_WIRE_SDA, PIN_WIRE_SCL);
+#if defined(NRF52805_XXAA)
```

Comment:
Could also consider `#if defined(NRF_TWIM1` here as well.

Reply:
I believe the NRF52805 has completely different handler names (TWIM0_TWIS0_TWI0_IRQn vs SPIM0_SPIS0_TWIM0_TWIS0_SPI0_TWI0_IRQn) which was part of the reason for using the board specific logic.

Reply:
Got it, sorry for not checking this out before!